db/chi_cache.sql

```diff
@@ -0,0 +1,5 @@
+CREATE TABLE `chi_cache` (
+  `key` VARCHAR(767),
+  `value` LONGTEXT,
+  PRIMARY KEY (`key`)
```

Comment:
The CHI driver itself manages to expire properly these entries and to ensure it doesn't get dirty.
So on on that side there is no gain except for the event we would need to deep debug CHI (which I  doubt we would need to do)